public/ng-views/maps.html

```diff
@@ -19,23 +22,23 @@
         <small>{{ user.name }}</small>
       </i>
     </span>
-  </span>
+  </span> -->
```

Comment:
this is where this commenting out gets dangerous, esp becuase angular relies on data binding inline to parse actions wrt to $scope